models/eos/ntp_global/eos_ntp_global.yml

```diff
@@ -0,0 +1,151 @@
+---
+module: eos_ntp_global
+short_description: Manages ntp resource module
+description: This module configures and manages the attributes of  ntp on Arista
+  EOS platforms.
+version_added: 3.1.0
+author: Gomathi Selvi Srinivasan (@GomathiselviS)
+notes:
+- Tested against Arista EOS 4.24.60M
+- This module works with connection C(network_cli). See the L(EOS Platform Options,eos_platform_options).
+options:
+   config:
+      description: A dictionary of ntp options
+      type: dict
+      suboptions:
+        authenticate:
+          description:
+          - Require authentication for NTP synchronization.
+          type: dict
+          suboptions:
+            enable:
+              description: Enable authentication for NTP synchronization.
+              type: bool
+            servers:
+              description: Authentication required only for incoming NTP server responses.
+              type: bool
+        authentication_key:
```

Comment:
In NX-OS (and possibly others too), it is possible to define multiple auth keys. Is it not the same for EOS?

Reply:
EOS supports multiple auth keys. I shall change authentication_key to a list of dict.